### DISCUSSION TITLE
How do I ssh into Raspberry Pi

### DISCUSSION THREAD

**Discussion by pjgoodall** [kovidgoyal/kitty#4281] [Q&A] [answered]
I need to ssh into my raspberry pi from Ubuntu 20.04. I realise kitty does not support rpi. But i still need to connect a terminal.
I would like to know a way of doing a normal terminal ssh connection avoiding the kitty stuff just for this connection.

**Comment by page-down** [comment, marked as answer]
Run `kitty +kitten ssh user@rpi-host` once to connect to rpi SSH, and later you can connect directly using `ssh`.
Or manually copy the file `xterm-kitty` to `~/.terminfo/x/xterm-kitty`.

https://sw.kovidgoyal.net/kitty/faq/

**Reply by pjgoodall** [reply]
Many Thanks.
Cheers...
